packages/now-static-build/test/fixtures/10a-gatsby-redirects/gatsby-node.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+exports.createPages = ({ actions }) => {
+  const { createRedirect } = actions;
+
+  createRedirect({
+    fromPath: "/permanent",
+    isPermanent: true,
+    toPath: "/"
+  });
+
+  createRedirect({
+    fromPath: "/not-permanent",
+    isPermanent: false,
+    toPath: "/"
+  });
+
+  createRedirect({
+    fromPath: "/custom-status",
+    statusCode: 404,
```

Comment:
Does status code 404 work? Perhaps this should be 307 instead?

Reply:
Yep it is how Gatsby shows example usage of `statusCode`.